librz/core/canalysis.c

```diff
@@ -3321,30 +3331,33 @@ RZ_API int rz_core_analysis_search_xrefs(RzCore *core, ut64 from, ut64 to, PJ *p
 
 	if (from == to) {
 		return -1;
-	}
-	if (from > to) {
-		eprintf("Invalid range (0x%" PFMT64x
-			" >= 0x%" PFMT64x ")\n",
-			from, to);
+	} else if (from > to) {
+		RZ_LOG_ERROR("Invalid range (0x%" PFMT64x " >= 0x%" PFMT64x ")\n", from, to);
 		return -1;
-	}
-
-	if (core->blocksize <= OPSZ) {
-		eprintf("Error: block size too small\n");
+	} else if (core->blocksize <= OPSZ) {
+		RZ_LOG_ERROR("block size is too small (blocksize <= %u)\n", OPSZ);
 		return -1;
 	}
+
 	ut8 *buf = malloc(bsz);
 	if (!buf) {
-		eprintf("Error: cannot allocate a block\n");
+		RZ_LOG_ERROR("cannot allocate a block\n");
 		return -1;
 	}
+
 	ut8 *block = malloc(bsz);
 	if (!block) {
-		eprintf("Error: cannot allocate a temp block\n");
+		RZ_LOG_ERROR("cannot allocate a temp block\n");
 		free(buf);
 		return -1;
 	}
+
 	rz_cons_break_push(NULL, NULL);
+
+	if (state->mode == RZ_OUTPUT_MODE_TABLE) {
+		rz_cmd_state_output_set_columnsf(state, "sXX", "type", "xref to", "xref at");
+	}
+
```

Comment:
```suggestion
	rz_cmd_state_output_set_columnsf(state, "sXX", "type", "xref to", "xref at");
```
The purpose of having `rz_cmd_state_output_set_columnsf` is exactly to avoid doing this kind of `if/else`.